editor/editor_translation.h

```diff
@@ -32,11 +32,14 @@
 #define EDITOR_TRANSLATION_H
 
 #include "core/string/ustring.h"
+#include "core/templates/list.h"
 #include "core/templates/vector.h"
 
 Vector<String> get_editor_locales();
 void load_editor_translations(const String &p_locale);
-void load_doc_translations(const String &p_locale);
 void load_property_translations(const String &p_locale);
+void load_doc_translations(const String &p_locale);
+void load_extractable_translations(const String &p_locale);
```

Comment:
This is starting to look like something that could be refactored in a single method, using an enum parameter to define whether it's editor, property, doc, or extractable.

But that can be for future rework.